bc_obps/service/tests/test_service_utils.py

```diff
@@ -0,0 +1,13 @@
+from django.test import TestCase
+from reporting.tests.utils.bakers import report_version_baker
+from service.utils.get_report_valid_year_from_version_id import get_report_valid_year_from_version_id
+from service.utils.constants import REPORT_VERSION_DATE_SUFFIX
+
+
+class TestServiceUtils(TestCase):
+    def test_get_report_valid_year_from_version_id(self):
+        report_version = report_version_baker()
+        assert (
+            get_report_valid_year_from_version_id(report_version.id)
+            == f"{report_version.report.reporting_year_id}{REPORT_VERSION_DATE_SUFFIX}"
```

Comment:
We would want to hardcode the object we expect here - see the other comment. Otherwise the test is tautological, we're (almost) doing "assert x = x" and changing dependencies won't make the test break.